src/app/page.tsx

```diff
@@ -28,6 +45,15 @@ export default function Home() {
         </div>
       </div>
 
+      <div className={style.container}>
+        <button className={style.leftButton} onClick={decrease}>-</button>
+        <div className={style.innerContainer}>
+            <p className={style.title}>Counter</p>
```

Comment:
since you are already importing the page.module.css stylesheet, there is no need to reference the style object directly and you can just say classname="leftButton" instead of className={style.leftButton}; makes it more concise

Reply:
turns out I was mistaken about this; what I suggested only works when using normal .css files and not .module.css files, so sorry about that